studio/package.json

```diff
@@ -24,8 +24,8 @@
     "eslint": "^8.6.0",
     "prop-types": "^15.7",
     "react": "^17.0",
+    "react-burger-menu": "^3.0.8",
```

Comment:
Why did you add this to the studio?

Reply:
was in the wrong folder when i npm installed